tests/testthat/test-import_instruments.R

```diff
@@ -65,7 +70,7 @@ target <- structure(
       "character"
     ))
   ),
-  row.names = c(NA, -5L), class = "data.frame"
+  row.names = c(NA, -5L), class = c("tbl_df", "tbl", "data.frame")
```

Comment:
adapt to the tibble that REDCapR returns.

Reply:
Need to double check the unit test still behaves but it makes good sense.